app/_src/guides/gateway-builtin.md

```diff
@@ -291,17 +334,16 @@ spec:
           certificates:
             - secret: my-gateway-certificate
         tags:
-          port: http-8080
-" | kubectl apply -f -
+          port: http-8080" | kubectl apply -f -
 ```
 
 Check the call to the gateway: 
-```shell
+```sh
 curl -X POST -v --insecure https://${PROXY_IP}:8080/increment
```

Comment:
```suggestion
curl -X POST -v --insecure "https://${PROXY_IP}:8080/increment"
```

For some reason for this one the copy paste adds backslash. I think the quotes should avoid this